messages/whatsapp/send-button-link.php

```diff
@@ -0,0 +1,53 @@
+<?php
+require_once __DIR__ . '../../config.php';
+require_once __DIR__ . '../../vendor/autoload.php';
+
+$keypair = new \Vonage\Client\Credentials\Keypair(
+    file_get_contents(VONAGE_APPLICATION_PRIVATE_KEY_PATH),
+    VONAGE_APPLICATION_ID
+);
+
+$client = new \Vonage\Client($keypair);
+
+$custom = [
+    "type" => "template",
+    "template" => [
+        "namespace" => WHATSAPP_TEMPLATE_NAMESPACE,
+        "name" => WHATSAPP_TEMPLATE_NAME,
+        "language" => ["code" => "en", "policy" => "deterministic"],
+        "components" => [
+            [
```

Comment:
Do the inner arrays here get converted to JSON objects? Note: the JSON for components uses both arrays _and_ objects. (Similar question for the other custom and template WhatsApp examples).

Reply:
The PHP array here is directly converted from the JSON example payload - so yeah what the client does in order to send the body is convert it to JSON